manim/mobject/types/vectorized_mobject.py

```diff
@@ -1023,6 +1049,50 @@ def point_from_proportion(self, alpha: float) -> np.ndarray:
 
             current_length += length
 
+    def proportion_from_point(
+        self,
+        point: typing.Iterable[typing.Union[float, int]],
+    ) -> float:
+        """Returns the proportion along the path of the :class:`VMobject`
+        a particular given point is at.
+
+        Parameters
+        ----------
+        point (typing.Iterable[float])
+            The Cartesian coordinates of the point which may or may not lie on the :class:`VMobject`
+
+        Returns
+        -------
+        float
+            The proportion along the path of the :class:`VMobject`.
+
+        Raises
+        ------
+        :exc:`ValueError`
+            If ``point`` does not lie on the curve.
+        :exc:`Exception`
+            If the :class:`VMobject` has no points.
+        """
+        self.throw_error_if_no_points()
+
+        nc = self.get_num_curves()
```

Comment:
Please rename to `num_curves` as `nc` isn't a very descriptive name

Reply:
Done.